test/alerts/repo_test.exs

```diff
@@ -54,6 +57,50 @@ defmodule Alerts.RepoTest do
       assert red_alert in alerts
       refute blue_alert in alerts
     end
+
+    test "returns an empty list of alerts when given an empty list of ids" do
+      assert [] = Repo.by_route_ids([], @now)
+    end
+  end
+
+  describe "diversions_by_route_id/2" do
+    test "returns only diversions for the given route_ids" do
+      # Setup
+      diversion = Factories.Alerts.Alert.build(:alert, effect: :shuttle)
+      non_diversion = Factories.Alerts.Alert.build(:alert, effect: :delay)
+
+      Store.update([diversion, non_diversion], nil)
```

Comment:
📓 We have to use this store until we convert the CMS code to using Mox for this.